indico/modules/events/papers/templates/management/overview.html

```diff
@@ -77,6 +104,30 @@
                 {% endif %}
             </div>
         </div>
+        <div class="section semantic-text">
+            <div class="icon icon-hammer"></div>
+            <div class="text">
+                <div class="label">{% trans %}Paper judging{% endtrans %}</div>
+                {% set judge_deadline = cfp.judge_deadline %}
```

Comment:
you could use `deadline` for all three places where you do this. I guess you do it to have something shorter, but in the trans line it's still quite long so it using just `deadline` would improve readability